README.md

```diff
@@ -10,6 +10,9 @@ Ambassador Edge Stack [![Build Status][build-status]][build-pages]
 [slack-url]:     https://a8r.io/slack
 [slack-join]:    https://img.shields.io/badge/slack-join-orange.svg
 
+> **Disclaimer**: This repository is in a read only state. We do not monitor issues or pull requests. Questions and issues should either be direct to the [Ambassador Labs community slack](https://a8r.io/slack) or the [Emissary-ingress](https://github.com/emissary-ingress/emissary/) repository. <br/> Information regarding specific edge-stack releases can be found in matching `rel/{release version}` branches. Example Edge-stack v3.2.0 information is at [rel/v3.2.0](https://github.com/datawire/edge-stack/tree/rel/v3.2.0).
```

Comment:
marketing consistency nit: 

Marketing refers to it as `Ambassador Edge Stack` but you used these `Edge-stack`, `edge-stack`. For consistency, we should use either the full name `Ambassador Edge Stack` or `Edge Stack`.

Reply:
> I see that some circle-ci jobs ran. I glanced at them and I think we want to gut those sooner rather than later so that we are not accidentally publishing anything. So, I think we will want to do that here in master and back in `apro.git` so that the mirrors don't run it.
> 
> I can also look on circle-ci but if I recall there is a way to unsubscribe a project so that it will stop monitoring a github repostitory. Thoughts?

I looked at them and then some of the other PRs. 

There should be a way to disable circle-ci for this repo....but just to remove unneeded code we should remove the circle ci folders in this PR and then do the same in apro